source/ui/Image.js

```diff
@@ -173,14 +180,15 @@
 		*/
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
+			var multiple= "url('"+ enyo.path.rewrite(src) +"'), url('"+ this.placeholder + "');";
```

Comment:
In order to properly support placeholder image assets, we will also need to rewrite the asset url (for the case where an SVG data URI is not specified).